File: ApiSuvesaPos/Negocio/Modelo/cxc/firmadocontado/autorizacionventa.vb
Namespace Modelo
    Public Class autorizacionventa
        Public Property Id As Integer
        Public Property Id_Factura As Integer
        Public Property Id_Cajero As String
        Public Property Id_Admin As String
        Public Property Cedula_Retira As String
        Public Property Nombre_Retira As String
        Public Property Cancelada As Boolean
        Public Property FechaCancelacion As Date
        Public Property IdApertura As Integer
        Public Property NumRecibo As Integer
        Public Property Telefono_Retira As String
        Public Property Express As Boolean
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/firmadocontado/autorizacionventa.cs
using System;

namespace Modelo
{
    public class autorizacionventa
    {
        public int Id { get; set; }
        public int Id_Factura { get; set; }
        public string Id_Cajero { get; set; }
        public string Id_Admin { get; set; }
        public string Cedula_Retira { get; set; }
        public string Nombre_Retira { get; set; }
        public bool Cancelada { get; set; }
        public DateTime FechaCancelacion { get; set; }
        public int IdApertura { get; set; }
        public int NumRecibo { get; set; }
        public string Telefono_Retira { get; set; }
        public bool Express { get; set; }
    }
}
